src/router.js

```diff
@@ -123,7 +121,7 @@ const routes = [
   },
   {
     name: ROUTE_NAME_SPECTATE,
-    path: '/spectate/:gameId',
+    path: '/spectate/:gameId?',
```

Comment:
You have some linting changes in here, looks like the only main change is the question mark for the conditional param.